Sources/SwiftSyntaxBuilder/BuildableNodes.swift.gyb

```diff
@@ -108,7 +108,13 @@ public struct ${type.buildable()}: ${base_type.buildable()}, ${type.expressible_
   /// Conformance to `${base_type.buildable()}`.
   public func build${base_type.base_name()}(format: Format, leadingTrivia: Trivia? = nil) -> ${base_type.syntax()} {
     let result = build${type.base_name()}(format: format, leadingTrivia: leadingTrivia)
-    return ${base_type.syntax()}(result).withLeadingTrivia(itemLeadingTrivia ?? .zero)
+    
+    let leadingTrivia =  [itemLeadingTrivia, result.leadingTrivia]
+      .compactMap { $0 }
+      .reduce(.zero, { $0 + $1 })
```

Comment:
Thanks, @ahoppen for pointing me out to 

> If you now stick that ImportDecl into a SourceFile and add a comment to the SourceFile

I've solved it by doing the following, what do you think?